### PR TITLE
Update CLI

### LINKED ISSUE #6: Update CLI
The CLI is outdated and does not work with the current version. We should also add logging verbosity to the CLI as an option.

### LINKED ISSUE #9: Add 'testing' instructions to Readme

### PR DESCRIPTION
**Pull request by lgrahl**
Closes #6, closes #9

- [x] Logging Verbosity Setting
- [x] Add Example to Readme
- [ ] Update Example Service File (@dbrgn)
- [x] Acceptable Test Coverage (@lgrahl)

### CODE REVIEW

**Comment by lgrahl** [comment]
I'm done here. Handing over to @dbrgn for the service example file.

**Comment by dbrgn** [comment]
CLI help says:

>   -k, --key PATH               Path to a or a hex-encoded private permanent key of the server (e.g.a NaCl private key).

But hex encoded strings don't seem to be accepted.

**Comment by dbrgn** [comment]
Updated the service file.

I don't really like that configuration data now needs to go into a service file. Can click also accept CLI options from configuration files?

File: saltyrtc/server/bin.py [hunk 1]
```diff
@@ -246,3 +246,7 @@ def main():
     finally:
         if obj['logging_handler'] is not None:
             obj['logging_handler'].pop_application()
+
+
+if __name__ == '__main__':
```
Comment (lgrahl) [inline saltyrtc/server/bin.py]: The line shouldn't be necessary as setuptools will generate a script that calls the `main` function above. Furthermore, running bin.py is probably not going to work as there are relative imports.

Comment (lgrahl) [inline saltyrtc/server/bin.py]:
```
python ../saltyrtc/server/bin.py
Traceback (most recent call last):
  File "../saltyrtc/server/bin.py", line 13, in <module>
    from . import __version__ as _version
SystemError: Parent module '' not loaded, cannot perform relative import
```

**Comment by lgrahl** [comment]
> But hex encoded strings don't seem to be accepted.

I'll look into it.

> I don't really like that configuration data now needs to go into a service file. Can click also accept CLI options from configuration files?

I found something: https://github.com/EverythingMe/click-config
But let's do that in another branch.

File: saltyrtc/server/bin.py (hunk 1 above)
Comment (dbrgn) [inline saltyrtc/server/bin.py]: This works, but only if the line is there:

    python -m saltyrtc.server.bin

I'm aware of the entry point script, but sometimes you want to call the module directly :)

**Pull request merged**